seqr/utils/search/utils.py

```diff
@@ -18,6 +18,34 @@ class InvalidSearchException(Exception):
     pass
 
 
+SEARCH_EXCEPTION_ERROR_MAP = {
+    InvalidIndexException: 400,
+    InvalidSearchException: 400,
+    elasticsearch.exceptions.ConnectionError: 504,
+    elasticsearch.exceptions.TransportError: lambda e: int(e.status_code) if e.status_code != 'N/A' else 400,
+}
```

Comment:
It may be better to define ES-specific things in the ES folder and use them here. For example, in the ES folder, we define `ES_SEARCH_EXCEPTION_ERROR_MAP`. And we define `SEARCH_EXCEPTION_ERROR_MAP = ES_SEARCH_EXCEPTION_ERROR_MAP` here. It is similar to the other constants below.